packages/prisma/migrations/20241218125130_add_optional_uuid_two_step_primary_key/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "SelectedCalendar" ADD COLUMN     "id" TEXT;
+
+-- Update all historical rows (id) to `gen_random_uuid`
+UPDATE "SelectedCalendar" SET "id" = gen_random_uuid() WHERE "id" IS NULL;
```

Comment:
Do we not want uid as the name here to match the Booking and EventTypeTranslation tables?

Reply:
Also because we are going to be migrating to use uuids all over and we will likely keep id (integer-based) during the migration.

Reply:
@emrysal and I talked and agreed that since this is new, we will go with id.